packages/core/ios/RNSentry.mm

```diff
@@ -162,6 +162,22 @@ - (SentryOptions *_Nullable)createOptionsWithDictionary:(NSDictionary *_Nonnull)
         return nil;
     }
 
+    // Exclude Dev Server and Sentry Dsn request from Breadcrumbs
+    NSString *dsn = [self getURLFromDSN:[mutableOptions valueForKey:@"dsn"]];
+    NSString *devServerUrl = [mutableOptions valueForKey:@"devServerUrl"];
+    sentryOptions.beforeBreadcrumb
+        = ^SentryBreadcrumb *_Nullable(SentryBreadcrumb *_Nonnull breadcrumb)
+    {
+        NSString *url = breadcrumb.data[@"url"] ?: @"";
+
+        if ([@"http" isEqualToString:breadcrumb.type]
+            && ((dsn != nil && [url containsString:dsn])
+                || (devServerUrl != nil && [url containsString:devServerUrl]))) {
```

Comment:
i: We could also only check the start of the string here too on the native side.
```suggestion
            && ((dsn != nil && [url hasPrefix:dsn])
                || (devServerUrl != nil && [url hasPrefix:devServerUrl]))
```

Reply:
Good idea. I already updated with c05c3c2679400cf285d5d3869df1eab5e3e61b9d after your JS comment ;)